### DISCUSSION TITLE
Share cloudlog dashbord for visitors

### DISCUSSION THREAD

**Discussion by ApkCreator** [magicbug/Cloudlog#2767] [General]
Everytime I look at my installed logbook hyperlink (https://volkmar-ines.de/cloudlog/), I will redirected to my login page. 
Is there a possibility to share the logbook dashbord for visitors?
Thanks a lot.
Best regards
Volkmar

**Comment by HB9HIL** [comment]
Hello

Go to the Station Logbooks and check if Visitor Log is enabled and which slug you have set.

You said here the Link with /cloudlog

This looks like the normal Main URL to me which redirects to the Login.

The Public Log usually looks like


/cloudlog/index.php/visitor/[your installed slug]

**Comment by ApkCreator** [comment]
Hello,
thanks for the quick response. 

But unfortunately the added link (https://volkmar-ines.de/cloudlog/index.php/visitor/) has no response. 

![GitHub](https://github.com/magicbug/Cloudlog/assets/13718777/32aa44c3-ed26-4e9b-862e-f9d20f9aa78c)

Thanks a lot.
Volkmar

**Reply by HB9HIL** [reply]
You have to paste in just the last part of the Link.. 

type in "log" and press Save.

The complete Link will be https://volkmar-ines.de/cloudlog/index.php/visitor/log then

**Comment by phl0** [comment]
> Is there a possibility to share the logbook dashbord for visitors?

I have implemented this using an apache redirect directive for the base URL. This requires a public slug to be set and Apache to be configured. Here is my config snippet:

```
RedirectMatch ^/$ https://cloudlog.m0abc.de
    /index.phpindex.php/publicSlug
```

**Comment by ApkCreator** [comment]
Hello, 
Thanks. All fine now.

I added my callsign and got the public link (https://volkmar-ines.de/cloudlog/index.php/visitor/DC8VS).

![GitHub_1](https://github.com/magicbug/Cloudlog/assets/13718777/c698ebce-9c02-4a7c-9954-85ef87a61519)

Thank you very much.
Volkmar

**Reply by HB9HIL** [reply]
works

**Reply by HB9HIL** [reply]
Please mark the question as answered :)

**Comment by ApkCreator** [comment]
Public link was created by add a continued folder name in the public link field inside the station logbook.
Thanks for the quick response.
